README.md

```diff
@@ -69,6 +69,17 @@ Install Kerberos client to enable `skt` authentication to Beaker:
 Install `skt` dependencies following its
 [README.md](https://github.com/RH-FMK/skt).
 
+Run tests
+---------
+
+For running all tests write down:
+
+    python -m pytest -v -s tests
+
+For running some specific tests you can do this as following:
```

Comment:
This needs to say either "you can do the following" or "you can do as follows". I would write this as "You can execute a specific test like this:". This can also be fixed in a separate PR.